tgui/packages/tgui/interfaces/TankCompressor.jsx

```diff
@@ -1,24 +1,31 @@
+import { toFixed } from 'common/math';
```

Comment:
> .jsx

You know what must be done...

jk if you need help with this just ask in discord. there's also `getShallowTypes(data)` if you want a quick list